aksel.nav.no/website/pages/templates/soknad-introside/intro-med-alert.tsx

```diff
@@ -33,28 +32,31 @@ function Example() {
     <Page footer={<Footer />}>
       <Header />
       <Page.Block width="text" gutters>
-        <VStack as="main" gap="8">
-          <Bleed
-            marginInline={{ lg: "24" }}
-            data-aksel-template="form-intropage-v2"
+        <VStack as="main" gap="8" data-aksel-template="form-intropage-v3">
+          <Stack
+            gap={{ xs: "3", lg: "6" }}
+            direction={{ xs: "column", lg: "row" }}
+            wrap={false}
+            position="relative"
           >
```

Comment:
I believe this could be simplified:
```suggestion
          <VStack gap="3">
```

Reply:
Nice call, updated all templates now. Can't really understand why this works without position relative on the VStack, but it works i guess so might just be missing something obvious.

Reply:
I wondered about the same, but I guess it's because we use `margin` and not `left`.